### PR TITLE
Added Ruby (bacon)

### PR DESCRIPTION
**Pull request by askl56**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Awesome TAP [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [<img src="https://testanything.org/images/tap.png" width="67" align="right">](https://testanything.org)
 
 
-> Useful resources for the [Test Anything Protocol](https://testanything.org)  
+> Useful resources for the [Test Anything Protocol](https://testanything.org)
```
Comment (sindresorhus) [inline readme.md:4]: Don't remove. Trailing whitespace is linebreak in markdown.

**Comment by sindresorhus** [comment]
Not sure about it. Doesn't look very maintained. It says it's just a clone of RSpec, so I don't see the point of it. The LOC argument is meaningless in this context.

**Comment by askl56** [comment]
It's the main TAP testing framework with Ruby. I understand your arguments though.

**Comment by sindresorhus** [comment]
How about https://github.com/rubyworks/rspec-ontap? I think that's better as you can just use the mainstream RSpec just with a third-party reporter.

**Pull request closed without merging**